week1/index.js

```diff
@@ -1,25 +1,60 @@
-// To run this file: node index.js
-// Then in your browser: http://localhost:8080
 var http = require('http');
```

Comment:
Thanks noted. It was more of copy/paste blind spot, I have updated the code accordingly